nton/constants.py

```diff
@@ -28,9 +28,10 @@ class Binaries:
 class Files:
     keys_home = (Path.home() / ".switch" / "prod.keys")
     keys_cwd = Path("./prod.keys").absolute()
-    keys = keys_cwd or keys_home
+    keys_exist = keys_home.is_file() or keys_cwd.is_file()
```

Comment:
Error to still have this line from a squash?

Reply:
you are completly right, we don't need it it's too early for my brain to work right :D